PULL_REQUEST_TEMPLATE

```diff
@@ -1,24 +1,32 @@
 Jira: [JIRA-TOKEN](https://ticketevolution.atlassian.net/browse/JIRA-TOKEN)
 
+> [!IMPORTANT]
+> Toggle to SQUASH AND MERGE when merging PRs to `staging` branch
```

Comment:
caps as emphasis isn't great, let's do bold?
```suggestion
> Toggle to **Squash and Merge** when merging PRs to `staging` branch
```

Reply:
**I VOTE FOR BOLD CAPS AND EVERYONE WINS** :trollface: 🤣